### PR TITLE
[ZIG-5127]Hotfix/adsrendertimeout error

### PR DESCRIPTION
**Pull request by 3akat**
## Proposed changes
- fixed ads-render-timeout crash

## Dependencies
- 

## Checklist
- [x] Tested locally
- [ ] Added new unit, integration or regression tests
- [ ] Updated docs

### CODE REVIEW

**Review by sambua — APPROVED** [review]
Good catch 👍

**Review by jimee02 — APPROVED** [review]

File: src/dynamics/video_player/player/player.js [hunk 1]
```diff
@@ -1257,7 +1257,10 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                  */
                 initAdsRenderFailTimeout: function() {
                     const renderTimeout = Number(this.get("adsrendertimeout"));
```
Comment (jimee02) [inline src/dynamics/video_player/player/player.js:1259]: fix: need a truthy check
```
Number(null) < 200
true
Number(undefined) < 200
false
```

Comment (jimee02) [reply src/dynamics/video_player/player/player.js:1259]: Check to see if `renderTimeout !== undefined` too

File: src/dynamics/video_player/player/player.js [hunk 2]
```diff
@@ -1257,7 +1257,10 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
...
-                    const repeatMicroseconds = 200;
+                    let repeatMicroseconds = 200;
+                    if (renderTimeout < repeatMicroseconds) {
+                        repeatMicroseconds = renderTimeout;
```
Comment (jimee02) [inline src/dynamics/video_player/player/player.js:1262]: Check to see if renderTimeout !== undefined too

File: src/dynamics/video_player/player/player.js (hunk 1 above)
Comment (jimee02) [reply src/dynamics/video_player/player/player.js:1259]: or you can use parseInt()
```
parseInt(undefined, 10) < 200
false
parseInt(null, 10) < 200
false
```

File: src/dynamics/video_player/player/player.js (hunk 2 above)
Comment (3akat) [reply src/dynamics/video_player/player/player.js]: Seems like this moving solve the issue https://github.com/betajs/betajs-media-components/blob/478af177345ce0008c660c9d52c0c82c9e232b94/src/dynamics/video_player/player/player.js#L1260-L1265

**Review by jimee02 — APPROVED** [review]

**Pull request merged**